ledger-core/virtual/callsummary/callsummary.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+//go:generate sm-uml-gen -f $GOFILE
+
+package callsummary
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/injector"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/tables"
+)
+
+type SummarySyncKey struct {
+	objectRef reference.Global
+}
+
+func BuildSummarySyncKey(objectRef reference.Global) SummarySyncKey {
+	return SummarySyncKey{objectRef: objectRef}
+}
+
+type SummarySharedKey struct {
+	pulseNumber pulse.Number
+}
+
+func BuildSummarySharedKey(pulse pulse.Number) SummarySharedKey {
```

Comment:
I don't like names `BuildSummarySharedKey` and `BuildSummarySyncKey `; I can't get difference of what they're doing outside of context. Can you, at least, add commentary? Or finding better name to describe their nature? 
Like `SummarySyncKey` is for temporary use, until `SummarySharedKey` is stored. Or something like that.

Reply:
Sure, I may add a comment. Why we have func BuildReportKey(object reference.Global) ReportKey?

Reply:
No, problem not in function for building key, but that i cannot understand difference between two that types of keys

Reply:
Added comments for keys